minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -86,6 +86,7 @@ public static LocalAlluxioMaster create(final String workDirectory) throws IOExc
     if (!Files.isDirectory(Paths.get(workDirectory))) {
       Files.createDirectory(Paths.get(workDirectory));
     }
+    Configuration.set(PropertyKey.MASTER_SAFEMODE_WAIT, "0ms");
```

Comment:
Move this to ConfigurationTestUtils#testConfigurationDefaults

Reply:
Done